content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst

```diff
@@ -15,11 +28,11 @@ Odoo automatically select how products are selected for orders:
    :stub-columns: 1
```

Comment:
This is out of scope of this PR, but want to note that this table does not format well on mobile. Maybe in another [IMP] PR this could be adjusted? I could see a card layout being useful here, the [Accounting and Invoicing](https://www.odoo.com/documentation/master/applications/finance/accounting.html) doc is a good example.